build/pipelines/templates/extensions-ci.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+- name: ExtensionDirectory
+  type: string
+  default: not-specified
+
+jobs:
+- job: "Build_And_Test_Windows"
+  pool:
+    vmImage: 'windows-latest'
```

Comment:
nit: I prefer to lock versions instead of latest here, it has bitten me in the past :)

Reply:
This was built from the existing definition. I can follow up with an update to include the SDK and worker as well.